README.md

```diff
@@ -57,95 +57,224 @@ Some of the skills/concepts this assignment aims to test (and build upon):
 * Routing (URL fragment based routing)
 
 ## API
-We can build API documentation using [Swagger](https://swagger.io/) or API Blueprint. 
+We can build API documentation using [Swagger](https://swagger.io/) or API Blueprint.
 A simple express JS API server can be built that is provided to students to allow them to develop offline. It could also be hosted somewhere to ease development when online?
 
 TODO: Someone will need to build a API server to support all of this client side functionality.
 
 The following specification is a WIP and only includes some endpoints.
 
+> POST `/login`
+
 ```json
-POST /login
 {
   "username": "string",
   "password": "string",
 }
+```
+
 returns:
+  - `200` OK for successful login, sets a cookie with a session token
+  - `403` for invalid credentials
+
+
+> POST `/signup`
 
-200 OK for successful login
-403 for invalid credentials
-POST /signup
+```json
 {
   "displayName": "string",
   "username": "string",
   "password": "string",
 }
-returns
+```
+
+> POST `/feed`
+
+```json
+{
+  "numItems": "integer",
+  "position": "string or null"
+}
+```
+
+returns:
+  - `200` OK for successful fetch, returns a list of feed items and a position token.
+    - position token allows a client to request 10 more posts from the last post requested onwards.
+  - `403` if user not logged in or invalid position supplied
+
+> POST `/like`
+
```

Comment:
PUT or POST. Also is it worth having this many API touchpoints. More common would be `/post/like` or `/post/{id}/like`